install.sh

```
@@ -0,0 +1,76 @@
+
+echo -e "${GREEN}RemindersSync Installation Script${NC}"
+echo "=================================="
+
+# Get the directory where the script is located
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd "$SCRIPT_DIR"
+
+echo -e "\n${YELLOW}Step 1: Building release version...${NC}"
+swift build -c release
+
+if [ $? -ne 0 ]; then
```

Shellcheck (0.10.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)